test/e2e/instrumentation/logging/stackdrvier/basic.go

```diff
@@ -107,6 +108,30 @@ var _ = instrumentation.SIGDescribe("Cluster level logging implemented by Stackd
 				err = utils.WaitForLogs(c, ingestionInterval, ingestionTimeout)
 				framework.ExpectNoError(err)
 			})
+
+			ginkgo.By("Checking that too long lines are trimmed", func() {
+				originalLength := 200 * 1024
+				longLine := strings.Repeat("a", originalLength)
+				pod, err := utils.StartAndReturnSelf(utils.NewRepeatingLoggingPod("synthlogger-4", longLine), f)
+				framework.ExpectNoError(err, "Failed to start a pod")
+
+				ginkgo.By("Waiting for logs to ingest")
+				c := utils.NewLogChecker(p, func(_ string, logEntries []utils.LogEntry) (bool, error) {
+					if len(logEntries) == 0 {
+						return false, nil
+					}
+					log := logEntries[0]
+					if log.JSONPayload != nil {
+						return false, fmt.Errorf("got json log entry %v, wanted plain text", log.JSONPayload)
+					}
+					if len(log.TextPayload) == originalLength {
+						return false, fmt.Errorf("got non-trimmed entry of length %d", len(log.TextPayload))
+					}
+					return true, nil
```

Comment:
Since you'll be documenting the trimmed entry prefix, want to also check that it's being prepended to trimmed entries (i.e., check that the entry starts with `[The entry was trimmed]`)?

Reply:
I thought about that, it created a dependency on the string constant between the code and configuration

OTOH, it's better to explicitly change/remove the test instead of debugging the problem b/c of lack of this check